printTextOrChangeText/view.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		
+	</head>
+	<body>
+		<h1>This is only a practic program</h1>
+		<h2>We will change html text in this program.</h2>
+		<p id="p1">This text will be change</p>
+		<button type="button" onclick="document.getElementById('p1').innerHTML='Text is chnaged'">click here!</button>
```

Comment:
please do not include javascript code in html, create separate file.

Reply:
done